Edits to text in API Conventions docs

dmeiss opened the pull request:
Minor edits to phrasing, punctuation, capitalization, and formatting.

dmeiss committed:
Edits to text in API Conventions docs

elasticmachine commented:
Pinging @elastic/es-core-infra

danielmitterdorfer commented:
@elasticmachine test this please

danielmitterdorfer commented on docs/reference/api-conventions.asciidoc:
```diff
@@ -526,7 +528,7 @@ If one of these quantities is large we'll print it out like 10m for 10,000,000 o
 
 [horizontal]
-``::   Single
+`s`::  Single
```
The `s` suffix is not recognized by Elasticsearch (see also https://github.com/elastic/elasticsearch/blob/869de3f160a727fedf2eb8dc41713e9dce24ba5f/server/src/main/java/org/elasticsearch/common/unit/SizeValue.java#L170-L197) so this needs to be an empty string.

danielmitterdorfer reviewed (changes requested):
Thanks for these fixes @dmeiss. The changes look mostly fine to me but I left one remark about an incorrect multiplier. Can you please remove the "s" multiplier?

dmeiss committed:
Made requested change on “s” multiplier

Deleted the “s”.

Is it OK that it then displays as two backtick characters? Or does it maybe need to be two single (or double) quotes to indicate an empty string, e.g. “”? 

I’m an editor by trade, not a coder — if that wasn’t already obvious. =).  Thanks for your patience.

dmeiss commented:
Made requested changes. Have a question though about how that empty string should be represented. It’s currently `` but I’m wondering if it should be “” instead.

danielmitterdorfer commented:
> Have a question though about how that empty string should be represented. It’s currently `` but I’m wondering if it should be “” instead.

On second thought we should just remove the entire line. I doubt that it helps anybody if we document that if no multiplier is specified, we treat the value as is. I think this is expected behavior anyway.

dmeiss commented:
I agree. Would you like me to go ahead & remove it?

dmeiss committed:
Removed "single" multiplier

dmeiss commented:
Went ahead & removed the "single" line

danielmitterdorfer reviewed (approved):
Thanks! Looks good to me.

danielmitterdorfer commented:
@elasticmachine test this please
